### PR TITLE
Feature/repo rpm

### PR DESCRIPTION
**Pull request by ghost**
spec file to create package installing PacketFence repo on Centos/RHEL

### CODE REVIEW

**Comment by extrafu** [comment]
Please also include the required documentation changes to handle the generated RPM.

**Comment by ghost** [comment]
doc included, please review then merge

File: addons/packages/packetfence-release.spec [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+BuildRoot: %{_tmppath}/%{real_name}-%{version}-%{rev}-root
+# disables the creation of the debug package for our setuid C wrapper
+%define debug_package %{nil}
+
+Packager: Inverse inc. <support@inverse.ca>
+Vendor: PacketFence, http://www.packetfence.org
+
+%description
+
+PacketFence release file. This package contains yum configuration
```
Comment (extrafu) [inline addons/packages/packetfence-release.spec:46]: the yum configuration

File: addons/packages/packetfence-release.spec [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+for the PacketFence RPM Repository.
```
Comment (extrafu) [inline addons/packages/packetfence-release.spec:47]: repository

File: addons/packages/packetfence-release.spec [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+%clean
+rm -rf $RPM_BUILD_ROOT
+
+%files -n %{real_name}
+%defattr(0755, root, root)
+%config /etc/yum.repos.d/packetfence.repo
+
+
+%changelog
+* Fri Apr 25 2014 Loick Pelet <lpelet@inverse.ca>
```
Comment (extrafu) [inline addons/packages/packetfence-release.spec:80]: Avoid putting "personal stuff".
=> Fri Apr 25 2014 Inverse inc. support@inverse.ca

**Pull request closed without merging**